subprojects/platform-native/src/main/groovy/org/gradle/nativeplatform/test/plugins/NativeBinariesTestPlugin.java

```diff
@@ -97,7 +97,9 @@ void attachBinariesToCheckLifecycle(ModelMap<Task> tasks, final ModelMap<NativeT
                 @Override
                 public void execute(Task checkTask) {
                     for (NativeTestSuiteBinarySpec testBinary : binaries) {
-                        checkTask.dependsOn(testBinary.getTasks().getRun());
+                        if (testBinary.isBuildable()) {
+                            checkTask.dependsOn(testBinary.getTasks().getRun());
+                        }
```

Comment:
This is what I'm working on. But there are complications so it is not
pushed yet.
Le 14 janv. 2016 22:57, "Chris Beams" <notifications@github.com> a écrit :

> In
> subprojects/platform-native/src/main/groovy/org/gradle/nativeplatform/test/plugins/NativeBinariesTestPlugin.java
> <https://github.com/gradle/gradle/pull/496#discussion_r49791548>:
>
> > @@ -97,7 +97,9 @@ void attachBinariesToCheckLifecycle(ModelMap<Task> tasks, final ModelMap<NativeT
> >                  @Override
> >                  public void execute(Task checkTask) {
> >                      for (NativeTestSuiteBinarySpec testBinary : binaries) {
> > -                        checkTask.dependsOn(testBinary.getTasks().getRun());
> > +                        if (testBinary.isBuildable()) {
> > +                            checkTask.dependsOn(testBinary.getTasks().getRun());
> > +                        }
>
> @melix <https://github.com/melix>, @lptr <https://github.com/lptr>,
> @bamboo <https://github.com/bamboo> you may want to take a look at this
> wrt the changes in 44a603d
> <https://github.com/gradle/gradle/commit/44a603d518dd04a85514fae6f55630236daae2d0>.
> Not sure whether we should be considering an #isBuildable guard there as
> well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gradle/gradle/pull/496/files#r49791548>.
>